core/templates/pages/library-page/library-page.constants.ts

```diff
@@ -27,13 +34,10 @@ export const LibraryPageConstants = {
     SEARCH: 'search'
   },
 
-  LIBRARY_PATHS_TO_MODES: {
-    '/community-library': 'index',
-    '/community-library/top-rated': 'group',
-    '/community-library/recently-published': 'group',
-    '/search/find': 'search'
-  },
+  LIBRARY_PATHS_TO_MODES: libraryPathToModes,
 
   SEARCH_EXPLORATION_URL_TEMPLATE:
     '/exploration/metadata_search?q=<query>'
 } as const;
+
+export type libraryPathToModesKeys = keyof typeof libraryPathToModes;
```

Comment:
Define this type in the component instead, by importing `LIBRARY_PATHS_TO_MODES`

Reply:
Done